third_party/amd/lib/TritonAMDGPUToLLVM/MemoryOpToLLVM.cpp

```diff
@@ -304,7 +304,7 @@ struct TransLocalLoadOpConversion
 
     // unpack i32 vectors and cast to native type
     if (bitwidth != 16) {
-      auto numElemsPerVec = 32 / bitwidth;
+      auto numElemsPerVec = std::max(32 / bitwidth, 1u);
```

Comment:
Where is this coming from? This path cannot be hit for bitwidth 64. Please carefully audit the changes and keep them to the bare minimum